packages/@lwc/babel-plugin-component/src/__tests__/index.spec.ts

```diff
@@ -58,28 +68,40 @@ function transform(source: string, opts = {}) {
         `apiVersion: 9999999`
     );
 
-    return code;
+    return {
+        code,
+        lwcErrors: (result.metadata as any)?.lwcErrors,
+    };
 }
 
 describe('fixtures', () => {
     testFixtureDir<LwcBabelPluginOptions>(
         {
             root: path.resolve(__dirname, 'fixtures'),
             pattern: '**/actual.js',
+            ssrVersion: 2,
         },
         ({ src, config }) => {
-            let result;
+            let code;
             let error;
+            let lwcErrors;
 
             try {
-                result = transform(src, config);
+                const result = transform(src, config);
+                code = result.code;
+                lwcErrors = JSON.stringify(normalizeLwcError(result.lwcErrors), null, 4);
```

Comment:
```suggestion
                lwcErrors = JSON.stringify(normalizeLwcError(result.lwcErrors), null, 4) + '\n';
```
This avoids needing to update `.prettierignore`.